pkgs/development/misc/cr16/newlib.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, texinfo, flex, bison, crossLibcStdenv, buildPackages }:
+
+let version = "2.0.0";
+in
+
+crossLibcStdenv.mkDerivation {
+  pname = "newlib";
+  inherit version;
+  src = fetchurl {
+    url = "ftp://sourceware.org/pub/newlib/newlib-${version}.tar.gz";
```

Comment:
This is a vanilla newlib src, so it shouldn't go in in a `cr16` directory. We might also have it already packaged?